public/index.html

```diff
@@ -221,10 +232,30 @@ <h1>Blog de QTSW</h1>
                 if (content) {
                     await addPost(content);
                     document.getElementById('content').value = '';
+                    // Zerar o contador de caracteres
+                    document.getElementById('charCount').textContent = '0/250 caracteres';
                 }
             });
 
             fetchPosts()
+
+
+            document.getElementById('content').addEventListener('input', function () {
+            const contentLength = this.value.length;
+            const maxLength = 100;
+            const charCount = document.getElementById('charCount');
+            
+            // Atualiza o contador de caracteres
+            charCount.textContent = `${contentLength}/${maxLength} caracteres`;
+
+            
+            if (contentLength >= maxLength) {
+                this.value = this.value.slice(0, maxLength); 
+            }
+        });
```

Comment:
Precisamos ajustar a identação do código:

```suggestion
            document.getElementById('content').addEventListener('input', function () {
                const contentLength = this.value.length;
                const maxLength = 100;
                const charCount = document.getElementById('charCount');
            
                // Atualiza o contador de caracteres
                charCount.textContent = `${contentLength}/${maxLength} caracteres`;

            
                if (contentLength >= maxLength) {
                    this.value = this.value.slice(0, maxLength); 
                }
            });
```